disputer/index.js

```diff
@@ -57,15 +58,20 @@ async function run(price, address, shouldPoll) {
 
 const Poll = async function(callback) {
   try {
-    if (!argv.address) {
+    if (!process.env.EMP_ADDRESS) {
       throw new Error("Bad input arg! Specify an `address` for the location of the expiring Multi Party.");
     }
     // TODO: Remove this price flag once we have built the pricefeed module.
-    if (!argv.price) {
+    if (!process.env.PRICE) {
       throw new Error("Bad input arg! Specify a `price` as the pricefeed.");
     }
 
-    await run(argv.price, argv.address, true);
+    let pollingDelay = 10_000; // default to 10 seconds, else use env value
+    if (!process.env.POLLING_DELAY) {
```

Comment:
Shouldn't this be `if(process.env.POLLING_DELAY)`? Because, if the env variable is set, then we should set `pollingDelay` to the env variable?

Reply:
ye, good point. that was an error. Have fixed this.